lib/dartdoc.dart

```diff
@@ -196,7 +196,8 @@ class DartDoc {
       ..sourceFactory = sourceFactory;
 
     if (packageMeta.isSdk) {
-      libraries.addAll(getSdkLibrariesToDocument(sdk, context));
+      libraries
+          .addAll(new Set()..addAll(getSdkLibrariesToDocument(sdk, context)));
```

Comment:
Would be awesome if collections supported a `distinct()` method, so we could just say `getSdkLibrariesToDocument(sdk, context).distinct()`.  That said, a set works in the meantime! 👍